pyfr/backends/base/makoutil.py

```diff
@@ -84,8 +84,10 @@ def _locals(body):
     # Extract the variable names
     lvars = [re.match(r'\s*(\w+)', v)[1] for v in decls]
 
+    keywords = {'if', 'int'}
```

Comment:
Is this needed?

Reply:
No, its for turbulence plugin. Will remove.